base/applications/network/tracert/tracert.cpp

```diff
@@ -376,29 +376,30 @@ RunTraceRoute()
         return false;
     }
 
-    BYTE SendBuffer[PACKET_SIZE];
-    ICMPV6_ECHO_REPLY ReplyBufferv6;
+    struct SendAndReplyStruct
+    {
+        BYTE SendBuffer[PACKET_SIZE];
+        ICMPV6_ECHO_REPLY ReplyBufferv6;
```

Comment:
If it is a hard requirement that this one is _directly_ behind the SendBuffer,
it would be nice to enforce that with a compile time assert (`C_ASSERT`)